viewing_party/party.py

```diff
@@ -0,0 +1,283 @@
+# *****WAVE 1******
+
+def create_movie(title, genre, rating):
```

Comment:
this is a good implementation. You can also do it without doing `dictionary_template = {}`

` if title and genre and rating:
        return {"title":title, "genre":genre, "rating":rating}
    else:
        return None`